activation/poet.go

```diff
@@ -492,7 +491,9 @@ func (c *poetService) Submit(
 		if err != nil {
 			return nil, fmt.Errorf("recertifying: %w", err)
 		}
-		return c.client.Submit(submitCtx, deadline, prefix, challenge, signature, nodeID, *auth)
+		return c.client.Submit(ctx, deadline, prefix, challenge, signature, nodeID, *auth)
+	case errors.Is(err, context.DeadlineExceeded):
+		logger.Warn("failed to submit challenge: probably submit challenge timeout is too short", zap.Error(err))
```

Comment:
NIT: the logged message isn't quite right

```suggestion
		logger.Warn("failed to submit challenge: PoET round already started", zap.Error(err))
```